odl/solvers/functional/functional.py

```diff
@@ -884,3 +885,44 @@ def convex_conj(self):
         """
         return self.functional.convex_conj.translated(
             self.linear_term)
+
+
+class FunctionalDefaultConvexConjugate(Functional):
+
+    """The ``Functional`` representing ``f^*(.)``.
```

Comment:
Also: the description should make sense of `f`. Like 
```
"""The `Functional` representing ``F^*``, the convex conjugate of ``F``."""
```
Abbreviate conj. or remove "convex" if it doesn't fit.